### ISSUE TITLE
Dropdown: Accessibility / gestures implementation broken

### ISSUE THREAD

**Issue by th0car** [seb-oss/green#1294]
### Bug already reported?

- [X] I confirm that I have checked if the bug already has been reported

### Component name

dropdown

### Description

When opening (by clicking) a GDS dropdown in an ios device in voiceover mode the close button automatically gets focus. 
From this state there is no way to traverse the list of options or even getting to the first one using gestures. It should also be possible to have the options read from top to bottom which is generally possible for dropdowns/selects. 

### Steps To Reproduce

1. Configure an iOS device to run voiceover.
2. Open a page with a dropdown instance (example: https://sebgroup.github.io/green/latest/core/iframe.html?args=&id=components-dropdown--page&viewMode=story
3. Open/Expand dropdown

### Current Behaviour

- Close button is automatically set in focus.
- There is no way to get to the first option in the list using gestures and thus no way to traverse the options
- There is no way to get the voiceover to read all the options 

### Expected Behaviour

As far as possible you should be able to use the same gestures as for a vanilla select - like:
- A simple right swipe should move focus to first option in list (if you not already there) to enable traversing the rest of the list using more right swipes
- A double down swipe should have the voiceover read all the options in the list
Whether the close button should always be in auto focus or not when dropdown is expanded I don't know. There _may_ be other alternatives that might enable first option to be in focus instead
